src/form.js

```diff
@@ -33,11 +34,22 @@ export default class SimpleteForm extends HTMLElement {
 		let field = this.searchField;
 		field.setAttribute("autocomplete", "off");
 
+		this.addAriaRoles();
+
 		let onQuery = debounce(this.queryDelay, this, this.onQuery);
 		this.addEventListener("input", onQuery);
 		this.addEventListener("change", onQuery);
 		this.addEventListener("simplete-suggestion-selection", this.onSelect);
 		field.addEventListener("keydown", this.onInput);
+		this.addEventListener("simplete-suggestion-toggle", this.onToggle);
+	}
+
+	addAriaRoles() {
```

Comment:
```suggestion
	addARIARoles() {
```

I'm not sure this warrants a separate method though. 🤷 (Just an observation; your call.)